<launch>

  <!-- USB cam node -->
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node"/>

  <!-- Detect Ball -->
  <param name="/detect_ball/h_min" type="int" value="0" />
  <param name="/detect_ball/h_max" type="int" value="41" />
  <param name="/detect_ball/s_min" type="int" value="27" />
  <param name="/detect_ball/s_max" type="int" value="201" />
  <param name="/detect_ball/v_min" type="int" value="166" />
  <param name="/detect_ball/v_max" type="int" value="255" />
  <node name="detect_ball_tuning" pkg="coconuts_master" type="detect_ball_tuning" />

</launch>